addon/helpers/sort-by.js

```diff
@@ -17,6 +21,15 @@ function sortDesc(key, a, b) {
   const aValue = get(a, key);
   const bValue = get(b, key);
 
+  if (typeof bValue == 'undefined') {
+    // keep bValue last
+    return -1;
+  }
+  if (typeof aValue == 'undefined') {
+    // put aValue last
+    return 1;
+  }
+
   if (aValue && bValue && aValue.toLowerCase && bValue.toLowerCase) {
```

Comment:
`aValue && bValue && ` can now be removed here as it was just to catch `undefined`. I think everything else can have `toLowerCase` check called on it.